third_party/nvfuser/csrc/contiguity.cpp

```diff
@@ -324,6 +324,48 @@ void OrderedIdInformation::handle(Swizzle2D* swizzle) {
   }
 }
 
+void OrderedIdInformation::handle(Resize* resize) {
```

Comment:
Do we need this change? Will setting resize's contiguity to `false` in `TensorDomain::getContiguousContiguity` solve the problem?

Reply:
This check is only for ordering of iter domains, it seems fine to extend it's analysis here consistently with Swizzle, even though swizzle can't be indexed contiguously.

Reply:
I think the contiguity of resize should be similar to the contiguity of expand prior to https://github.com/csarofeen/pytorch/pull/2517, the dimension of resize and the dimension before resize should never be contiguous. And yeah, I am OK with the change here, just asking some general question.

Reply:
I think what I did here is to just follow the same logic as is done for the other expressions. It's not contiguously indexed, so this isn't technically necessary.